+Prefs/+Inputs/ButtonField.m

```diff
@@ -13,23 +9,13 @@
         function [obj,height_px,label_width_px] = make_UI(obj,pref,parent,yloc_px,width_px,margin)
             % Here, widths will all be taken care of in adjust_UI
             [obj,height_px,label_width_px] = make_UI@Prefs.Inputs.LabelControlBasic(obj,pref,parent,yloc_px,width_px);
-            set(obj.ui,'TooltipString',pref.value);
-            if isempty(pref.value)
-                set(obj.ui,'String',obj.empty_string);
-            end
+            set(obj.ui,'String',pref.string);
```

Comment:
does a pref have a string property?

Reply:
I'm definitely open to suggestions on this. `Prefs.Button` has a `string` field, and I want the user to have an option of both giving it a `name` (displayed left of the button) and labeling the button itself. The alternative is to display the `unit` string on the button even though this is slightly profane. I think there are other inputs that only work with specific prefs also.